ansible/group_vars/all

```diff
@@ -1058,6 +1058,59 @@ rundeck:
     ttl: 60
     overwrite: yes
 
+# Spark Prototype configuration
+spark_proto:
+  skip: true
+  elastic_load_balancer:
+    name: "{{ vpc_short_name }}-sparkpto-elb"
+    connection_draining_timeout: 600
```

Comment:
Just by curiosity, why is it 60 for all the others except impac, connec and sparkpost ?